synapse/cores/ram.py

```diff
@@ -175,6 +180,24 @@ def _getSizeByProp(self, prop, valu=None, mintime=None, maxtime=None):
 
         return len(rows)
 
+    def _getCoreType(self):
+        return 'ram'
+
+    def _getBlobValu(self, key, default):
+        v = self._blob_store.get(key, default)
+        if v is s_common.novalu:
+            raise s_common.NoSuchName(name=key, mesg='Blob store has no such key present.')
```

Comment:
Mostly though, they will know if the key was present or not by the convention of what they expect.  For example, most of those keys will never have the actual *value* None, so if you get None back, you know it was not present ( and if you do, you can likely treat them the same anyway )